src/pages/_app.tsx

```diff
@@ -1,9 +1,24 @@
 import type { AppProps } from "next/app";
+import { NextUIProvider, globalCss } from "@nextui-org/react";
 
-import "@styles/globals.css";
+// import "@styles/globals.css";
+import { lightTheme } from "../styles/theme";
+
+//Global styles can be added here
+const globaStyles = globalCss({
```

Comment:
Isn't this redundant considering there's a base `theme.ts` file as well (I'm genuinely asking, correct me if I'm wrong)?

Reply:
Theme file doesn't set global styles, it basically set themes and its brand colors which we can use throughout our code. And about global styles such as css reset and fonts etc, we can either use this or our styles/global.css. it's upto us and I would like to know which one would you prefer.

Reply:
Let's use a `globals.css` file instead then? It should help us refactor the source code with better accuracy when needed in the future. And having the global styles in its own file will unclutter the `_app.tsx` file as the project grows over time.

Reply:
Got it, will update that as well.